mmdet/models/backbones/res2net.py

```diff
@@ -0,0 +1,319 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import build_conv_layer, build_norm_layer
+
+from ..builder import BACKBONES
+from .resnet import Bottleneck as _Bottleneck
+from .resnet import ResNet
+
+
+class Bottle2neck(_Bottleneck):
+    expansion = 4
+
+    def __init__(self,
+                 inplanes,
+                 planes,
+                 scale=4,
```

Comment:
Better to rename it to `scales` or `num_scales`

Reply:
I have changed the term 'scale' to 'scales'.